build/test/TestPackageProjects.targets

```diff
@@ -74,14 +74,14 @@
         <Version>2.0.0</Version>
         <Clean>True</Clean>
       </BaseTestPackageProject>
-      <BaseTestPackageProject Include="TestAssets/TestPackages/dotnet-outputsframeworkversion/dotnet-outputsframeworkversion-netcoreapp1.0">
+      <!--<BaseTestPackageProject Include="TestAssets/TestPackages/dotnet-outputsframeworkversion/dotnet-outputsframeworkversion-netcoreapp1.0">
```

Comment:
Did you remove these commented code in later changes? We should just remove them.